docs/krel/gcbmgr_generate.md

```diff
@@ -0,0 +1,51 @@
+# generate <!-- omit in toc -->
+
+`generate` is a subcommand of `krel gcbmgr` for Release Managers to build a markdown table that contains all the jobs and necessary information to complete the Cut Release GitHub Issue.
+
+`krel gcbmgr generate` collect all the jobs that run in `google cloudbuild` for a specific release in a specific date. The intention of this subcommand is to automate the taks of filling the data in the GitHub Issue by building the entire table when all the jobs ran, instead of doing one by one manually.
```

Comment:
```suggestion
`krel gcbmgr generate` collect all the jobs that run in `google cloudbuild` for a specific release in a specific date. The intention of this subcommand is to automate the tasks of filling the data in the GitHub Issue by building the entire table when all the jobs ran, instead of doing one by one manually.
```

Reply:
I think its clear just found this little typo :point_up: